src/components/sdk/DeviceSelection/docs/VideoFilterSelection.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoFilterSelection from '../CameraSelection/VideoFilterSelection';
+
+<Meta title="SDK Components/DeviceSelection/Camera/VideoFilterSelection" />
+
+# VideoFilterSelection
+
+The `VideoFilterSelection` component renders a `Select FormField` to select a video filter. Currently, the only option is background blur.
```

Comment:
Shouldn't the document say `Check FormField` now?

Reply:
yup - i changed that.

Reply:
Thanks :)